### PR TITLE
Adds correct ID's and baseURL for OktaAuth

### PR DESCRIPTION
**Pull request by iOSjess**
Revised OktaAuth info to the new info provided by Lambda staff.

### CODE REVIEW

**Review by claywatkins — COMMENTED** [review]
Thank you for making this change, weird how it needed to be optional!

File: labs-ios-starter/Model Controller/ProfileController.swift [hunk 1]
```diff
@@ -14,13 +14,13 @@ class ProfileController {
...
-                            clientID: "0oalwkxvqtKeHBmLI4x6",
-                            redirectURI: "labs://scaffolding/implicit/callback")
+                            clientID: "0oa18is3355KlyP5C4x7",
+                            redirectURI: "labs://cityspire/implicit/callback")
     
     private(set) var authenticatedUserProfile: Profile?
     private(set) var profiles: [Profile] = []
     
-    private let baseURL = URL(string: "https://labs-api-starter.herokuapp.com/")!
+    private let baseURL = URL(string: "https://cityspire-a-be.herokuapp.com/")! // https://labs-api-starter.herokuapp.com/
```
Comment (claywatkins) [inline labs-ios-starter/Model Controller/ProfileController.swift]: Thank you for making these changes, looks good to me.

File: labs-ios-starter/Model Controller/ProfileController.swift [hunk 2]
```diff
@@ -14,13 +14,13 @@ class ProfileController {
...
```
Comment (johnnydhicks) [inline labs-ios-starter/Model Controller/ProfileController.swift]: You can get rid of the comment here, that URL isn't useful

File: labs-ios-starter/Model/Profile.swift [hunk 3]
```diff
@@ -10,9 +10,9 @@ import UIKit
 
 struct Profile: Codable {
     
-    let id: String
-    let email: String
-    let name: String
+    let id: String?
+    let email: String?
+    let name: String?
```
Comment (johnnydhicks) [inline labs-ios-starter/Model/Profile.swift:15]: Remove the optionality here. I only told you to make them optional for the sample endpoint, but your deployed endpoint shouldn't allow any of these to be nil

**Review by johnnydhicks — CHANGES_REQUESTED** [review]

File: labs-ios-starter/Model Controller/ProfileController.swift (hunk 2 above)
Comment (iOSjess) [reply labs-ios-starter/Model Controller/ProfileController.swift]: Oh, yeah. Sorry.

File: labs-ios-starter/Model/Profile.swift (hunk 3 above)
Comment (iOSjess) [reply labs-ios-starter/Model/Profile.swift:15]: Ok, will do.

**Review by johnnydhicks — APPROVED** [review]

**Pull request merged**